unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridController.cs

```diff
@@ -185,7 +185,8 @@ private async UniTask<int> RequestWearablesAndShowThem(int page, CancellationTok
                 currentWearables = wearables.Select(ToWearableGridModel)
                                             .ToDictionary(item => item.WearableId, model => model);
 
-                view.SetWearablePages(page, (totalAmount / PAGE_SIZE) + 1);
+                view.SetWearablePages((totalAmount + PAGE_SIZE - 1) / PAGE_SIZE);
```

Comment:
what is the gain of separating the operation in two methods?

Reply:
Sorry, you're right... it was not needed. I had some problems to reset the page selector to the first page each time we change the filters and I finally left the separated flow. I have unified them again. Done here: https://github.com/decentraland/unity-renderer/pull/5119/commits/defc148a43876106380609d549752438a85841f9